### DISCUSSION TITLE
Is there any request body documentation?

### DISCUSSION THREAD

**Discussion by araujo-filipe** [AbsaOSS/spline-spark-agent#310] [Q&A] [answered]
Hey guys,

Is there any available documentation on the request body sent to each of the endpoints?

Thank you for all the great work!

Best,
João

**Comment by wajda** [comment, marked as answer]
Hi João,
There is a Swagger definition - [producer.api.v1.1.json](https://github.com/AbsaOSS/spline-spark-agent/blob/develop/core/src/main/resources/producer.api.v1.1.json) (Kafka endpoint is fed with the same JSON structure as the REST endpoint)
The Swagger is not too descriptive though, so you may also want to take a look at the [domain model](https://github.com/AbsaOSS/spline/blob/gh-pages/docs/Persistent%20model.pdf) docs for more details.
